src/index.html

```diff
@@ -5,10 +5,40 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="moyo_1.png"
+        alt="MoyoLogo"
+        class="logo__img">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="index.html" class="nav__link is-active">APPLE</a></li>
```

Comment:
- Add empty lines between multiline sibling blocks of HTML
<img width="1036" alt="Screenshot 2023-10-13 at 13 53 08" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/dd74c615-73b6-4c0c-8475-cbf4a1070eba">